python.d/go_expvar.chart.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+# Description: go_expvar netdata python.d module
+# Author: Jan Kral (kralewitz)
+
+from __future__ import division
+from base import UrlService
+import json
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
+priority = 60000
+retries = 60
+
+
+MEMSTATS_CHARTS = {
+    'memstats.heap': {
+        'options': ['heap', 'memory: size of heap memory structures', 'kB', 'memstats', 'expvar.memstats.heap', 'line'],
+        'lines': [
+            ['memstats.heap_alloc', 'alloc', 'absolute', 1, 1024],
+            ['memstats.heap_inuse', 'inuse', 'absolute', 1, 1024]
```

Comment:
Oh yeah, that totally makes sense. I myself use InfluxDB that is indeed dot-sensitive so that would probably bite me very soon :)